<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitialConfig</class>
 <widget class="QWidget" name="InitialConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Initial Config.</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="configDataCoordinates_label">
     <property name="text">
      <string>Data coordinates:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="FieldChooserWidget" name="configDataCoordinates_fieldChooser">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="configModelCoordinates_label">
     <property name="text">
      <string>Model coordinates:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="FieldChooserWidget" name="configModelCoordinates_fieldChooser">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="configModelFitGroup_label">
     <property name="text">
      <string>Model fit group:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="FieldChooserWidget" name="configModelFitGroup_fieldChooser">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="configFibreOrientation_label">
     <property name="text">
      <string>Fibre orientation:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="FieldChooserWidget" name="configFibreOrientation_fieldChooser">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Field supplying Euler angles to rotate local 'fibre' axes on which strain and curvature penalties are applied. Clear to apply on global x, y, z axes. Required for fitting 2D meshes with 3 coordinate components.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="configDiagnosticLevel_label">
     <property name="text">
      <string>Diagnostic level:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="configDiagnosticLevel_spinBox">
     <property name="maximum">
      <number>2</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FieldChooserWidget</class>
   <extends>QComboBox</extends>
   <header>cmlibs/widgets/fieldchooserwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
